fireTS/models.py

```diff
@@ -109,7 +109,13 @@ def predict(self, X, y, step=1):
 
         ypred = np.concatenate([np.empty(step) * np.nan, yhat])[0:len(y)]
         return ypred
-
+    
+    
+    def forecast(self, X):
+        """
```

Comment:
Please take the doc string of `predict` method as a template to write up the doc string for forecast.